### DISCUSSION TITLE
How up to date is your weblate?

### DISCUSSION THREAD

**Discussion by DenuxPlays** [keycloak/keycloak#34928] [Q&A] [answered]
Hey,

we want to contribute our translations to keycloak, sadly weblate seems to miss a lot of languages that are already in keycloak.
Is the weblate being used and how up to date is it?

**Comment by jonkoops** [comment, marked as answer]
Not all translations are in Weblate for a reason, see the the [translations documentation](https://github.com/keycloak/keycloak/blob/main/docs/translation.md)

**Reply by DenuxPlays** [reply]
Okay thats sad

Almost all of them do not know how to use GitHub so they wont be able to contribute to them.

**Reply by jonkoops** [reply]
If you read the document you will see that they can, but only if we have maintainers to support them.

**Reply by DenuxPlays** [reply]
Yes I know but I doubt that we can find two people.
